kubectl-cloudflow/cmd/deploy.go

```diff
@@ -96,16 +97,21 @@ You can update the credentials with the "update-docker-credentials" command.
 func (opts *deployOptions) deployImpl(cmd *cobra.Command, args []string) {
 	version.FailOnProtocolVersionMismatch()
 
-	imageRef := args[0]
-	imageReference, err := parseImageReference(imageRef)
+	crFile := args[0]
+	crString, err := util.GetFileContents(crFile)
```

Comment:
minor to cleanup: can probably give some nicer feedback to the user if the file does not exist, has bad format etc, later.

Reply:
done ..